debian/changelog

```diff
@@ -1,3 +1,10 @@
+tiqit (1.0.9-1) trusty; urgency=low
```

Comment:
Given that 1.0.8 isn't in master branch I reckon we could slip this into 1.0.8

You'll need to downgrade your previous changes below back to 1.0.8